docker/daemon.go

```diff
@@ -413,7 +412,7 @@ func initRouter(s *apiserver.Server, d *daemon.Daemon) {
 		image.NewRouter(d, decoder),
 		systemrouter.NewRouter(d),
 		volume.NewRouter(d),
-		build.NewRouter(dockerfile.NewBuildManager(d)),
```

Comment:
it's still used in https://github.com/docker/docker/pull/21636/files#diff-9466c26908f68c0d8d73c7a0e23fae9dR29